### ISSUE TITLE
 Multi-site Submissions

### ISSUE THREAD

**Issue by giorgiopogliani** [verbb/formie#149]
As of discussion on #84 I'am opening this issue for multi-site submissions support. 

It would nice to filter the submissions based on which site the form was sent like any other entry. Hence, the export feature should be improved to export on per site basis.

**Comment by engram-design** [comment]
Thanks for your PR, after some proper time looking into it, I actually removed `originSiteId` in favour of the submission element's `siteId` (no need for a record attribute). Sorry to mess you around with that. There were also a few other things I needed to tweak like ensuring submission elements aren't propagated across all sites. They should be applicable to just one site.

Added in [1.2.20](https://github.com/verbb/formie/releases/tag/1.2.20)
